src/pages/Login.jsx

```diff
@@ -18,9 +24,34 @@ const Login = () => {
           </Box>
           <Box component="form" sx={{ display: "flex", flexDirection: "column", gap: 2, marginY: 5 }}>
             <Typography variant="body2">Email</Typography>
-            <TextField variant="outlined" onChange={(event) => setEmail(event.target.value)} value={email} data-testid="email-field" inputProps={{ "data-testid": "email-content" }} />
+            <TextField placeholder="Email" variant="outlined" onChange={(event) => setEmail(event.target.value)} value={email} data-testid="email-field" inputProps={{ "data-testid": "email-content" }} />
             <Typography variant="body2">Password</Typography>
-            <TextField type="password" variant="outlined" data-testid="password-field" inputProps={{ "data-testid": "password-content" }} onChange={(event) => setPassword(event.target.value)} value={password} />
+            <div style={{ position: "relative" }}>
+              <TextField
+                type={showPassword ? "text" : "password"}
+                placeholder="Password"
+                onChange={(e) => setPassword(e.target.value)}
+                value={password}
+                data-testid="password-field"
+                inputProps={{ "data-testid": "password-content" }}
+                style={{ width: "100%" }}
+              />
+              <IconButton
```

Comment:
I changed the div to IconButton tag since we're using MUI so I think it would be better to follow the rules.